File: PHP/payment.php
<?php
class Payment {
    public $id;
    public $amount;

    public function __construct($id, $amount){
        $this->id = $id;
        $this->amount = $amount;
    }
    public function PrintDataPayment(){
        echo "Id: $this->id; Amount: $this->amount <br/>";
    }
}

File: Java/Payment.java
class Payment {
    Integer id;
    Integer amount;

    public Payment(Integer id, Integer amount){
        this.id = id;
        this.amount = amount;
    }
    void printDataPayment() {
        System.out.println("Id: " + id + "; Amount: " + amount);
    }
}